lib/ncr/reporter.rb

```diff
@@ -43,5 +62,34 @@ def self.budget_proposals(type, timespan)
               .where('created_at > ?', timespan)
               .select { |pr| pr.client_data.expense_type == type }
     end
+
+    # rubocop:disable all
+    def self.proposals_tier_one_pending_sql
+      tier_one_sql = User.sql_for_role_slug('BA61_tier1_budget_approver', 'ncr')
+
+      approver_sql = <<-SQL.gsub(/^ {8}/, '')
```

Comment:
FYI that you can use [`strip_heredoc`](http://api.rubyonrails.org/classes/String.html#method-i-strip_heredoc). Not sure that it's necessary for SQL, though...